spring-session-data-mongodb/src/main/java/org/springframework/session/data/mongo/JacksonMongoSessionConverter.java

```diff
@@ -16,11 +16,6 @@
 
 package org.springframework.session.data.mongo;
 
-import java.io.IOException;
```

Comment:
Please only make changes that are necessary for this PR. Note that this breaks checkstyle. You can run `./gradlew check --continue`

Reply:
Thank you for the reminder. I have resubmitted the code as requested.